examples/pcb-aoi/incremental_learning_bench/benchmarkingjob.yaml

```diff
@@ -8,13 +8,18 @@ benchmarkingjob:
   # the file format supports yaml/yml;
   testenv: "./examples/pcb-aoi/incremental_learning_bench/testenv/testenv.yaml"
 
-  # test algorithm configuration files; list type;
-  algorithms:
-    # algorithm name; string type;
-    - name: "fpn_incremental_learning"
-      # the url address of test algorithm configuration file; string type;
-      # the file format supports yaml/yml
-      url: "./examples/pcb-aoi/incremental_learning_bench/testalgorithms/fpn/fpn_algorithm.yaml"
+  # the configuration of test object
+  test_object:
+    # test type; string type;
+    # currently the option of value is "algorithms",the others will be added in succession.
```

Comment:
There exists different types of test objects in ianvs benchmaking. At present, we first support algortihms as test objects. Espect algorithms, systems or even datasets can also be test objects, which will be included in future versions.